src/plugins/blocksmith/components/blocksmith/index.jsx

```diff
@@ -347,6 +347,69 @@ class Blocksmith extends React.Component {
     updateBlocks.clearCache();
   }
 
+  /**
+   * Given a canvas block, creates a DraftJs block (with data) for said block at the active block.
+   *
+   * @param {*} canvasBlock                - a triniti canvas block
+   * @param {boolean} shouldSelectAndStyle - whether or not to select and style the new block
+   */
+  handleAddCanvasBlock(canvasBlock, shouldSelectAndStyle = false) {
```

Comment:
@aloncohenn i get a lint error if you move this fn, do you?

<img width="808" alt="Screen Shot 2021-02-18 at 8 42 42 AM" src="https://user-images.githubusercontent.com/6315388/108390242-51cfc900-71c5-11eb-9ad4-3e10eda91329.png">

Reply:
i'm getting a lint error when i move it back to where it was, i moved again